### PR TITLE
rados: include generic options in usage message

### PR DESCRIPTION
**Pull request by rs-fabrica**
Just including generic options in rados usage message.

### CODE REVIEW

**Comment by rs-fabrica** [comment]
AFAICT the failed check in not related to the changes I made. It seems to be something related to the Jenkins build configuration.

Please advise.

**Comment by varshar16** [comment]
jenkins retest this please

File: src/tools/rados/rados.cc [hunk 1]
```diff
@@ -290,7 +290,7 @@ int trunc([[maybe_unused]] IoCtx& io_ctx, const std::string& oid, const uint64_t
   if (use_striper)
     return striper().trunc(oid, offset);
 #endif
- 
+
```
Comment (tchaikov) [inline src/tools/rados/rados.cc:298]: in general, i'd suggest avoid massive reformatting, unless you happen to be changing this area.

File: src/tools/rados/rados.cc [hunk 2]
```diff
@@ -237,11 +237,13 @@ void usage(ostream& out)
...
 "   --run-length                     total time (in seconds)\n"
-"   --offset-align                   at what boundary to align random op offsets"
+"   --offset-align                   at what boundary to align random op offsets\n"
+"\n"
 "CACHE POOLS OPTIONS:\n"
 "   --with-clones                    include clones when doing flush or evict\n"
+"\n"
 "OMAP OPTIONS:\n"
-"    --omap-key-file file            read the omap key from a file\n";
+"    --omap-key-file file            read the omap key from a file\n";;
```
Comment (tchaikov) [inline src/tools/rados/rados.cc]: please remove the trailing ";".

File: src/tools/rados/rados.cc [hunk 3]
```diff
@@ -243,7 +243,10 @@ void usage(ostream& out)
 "   --with-clones                    include clones when doing flush or evict\n"
 "\n"
 "OMAP OPTIONS:\n"
-"    --omap-key-file file            read the omap key from a file\n";;
+"    --omap-key-file file            read the omap key from a file\n"
+"\n"
```
Comment (tchaikov) [inline src/tools/rados/rados.cc:247]: i am not sure what 

> Fixed the formatting of the last options in rados usage message

means. what's wrong with the last options?

File: src/tools/rados/rados.cc (hunk 2 above)
Comment (tchaikov) [reply src/tools/rados/rados.cc]: could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

"rados_generic_options_usage_message" is not an ideal prefix in this context.

Comment (tchaikov) [reply src/tools/rados/rados.cc]: "fixing formating" is way too vague. please be more specific. and please fix the type: s/formating/formatting/

**Comment by rs-fabrica** [comment]
I have just substituted the previous commits with new ones that hopefully fixes all issues mentioned.

Please let me know if there is anything else to change.

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2020-07-25_02:41:59-rados-wip-kefu-testing-2020-07-24-2327-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-07-25_13:22:44-rados-wip-kefu-testing-2020-07-24-2327-distro-basic-smithi/

**Comment by cbodley** [comment]
does this need backports? there's a report about the issue on nautilus at https://tracker.ceph.com/issues/53743

**Pull request merged**